rln/src/protocol.rs

```diff
@@ -49,6 +49,7 @@ pub struct RLNProofValues {
     // Public Inputs:
     pub x: Fr,
     pub external_nullifier: Fr,
+    pub message_id: Fr,
```

Comment:
isn't message_id a private input?
https://github.com/Rate-Limiting-Nullifier/circom-rln/blob/b40dfa63b7b1248527d7ab417d0d9cf538cad93a/circuits/rln.circom#L40

Reply:
Added corresponding comment

Reply:
the struct RLNProofValues should only contain the data required to verify a proof, right? can we remove message_id from this?

Reply:
Removed, as requested